interface/main/calendar/add_edit_event.php

```diff
@@ -1602,8 +1696,14 @@ function isRegularRepeat($repeat){
   <td nowrap>
 
 <?php
-generate_form_field(array('data_type'=>1,'field_id'=>'apptstatus','list_id'=>'apptstat','empty_title'=>'SKIP'), $row['pc_apptstatus']);
+if($_GET['group']!=true) {
+    generate_form_field(array('data_type' => 1, 'field_id' => 'apptstatus', 'list_id' => 'apptstat', 'empty_title' => 'SKIP'), $row['pc_apptstatus']);
+}
+else{
+    generate_form_field(array('data_type' => 1, 'field_id' => 'apptstatus', 'list_id' => 'groupstat', 'empty_title' => 'SKIP'), $row['pc_apptstatus']);
+}
```

Comment:
Potentially critical issue:
This will cause many potential downstream issues by using a different set of apptstatus elements for group vs non-group in the appointment. What are the actual differences here in these 2 different lists? Then can figure out best way to deal with this.